src/midi/midi_mt32.h

```diff
@@ -34,14 +34,15 @@
 
 #define MT32EMU_API_TYPE 3
 #include <mt32emu/mt32emu.h>
-#if MT32EMU_VERSION_MAJOR != 2 || MT32EMU_VERSION_MINOR < 1
-#error Incompatible mt32emu library version
-#endif
 
 #include "mixer.h"
 #include "rwqueue.h"
 #include "soft_limiter.h"
 
+static_assert(MT32EMU_VERSION_MAJOR > 2 ||
+                      (MT32EMU_VERSION_MAJOR == 2 && MT32EMU_VERSION_MINOR >= 5),
```

Comment:
Suggestion: maybe we should implement and submit upstream macro `MT32EMU_VERSION_ATLEAST`, which would work the same as `SDL_VERSION_ATLEAST` (see `SDL_version.h`).